modify.md

```diff
@@ -54,7 +54,27 @@ class $modify(MenuLayer) {
 
 (or you could be questionable and do `this->MenuLayer::init()` but who am I to judge)
 
-This pattern is quite common inside cocos2d, and therefore in GD too. Here, we modify the `MenuLayer::init` so that it will create our label inside the layer after running the original function. If `MenuLayer::init` returns false something has gone horribly wrong and GD will probably crash but it is important to stick to the idioms for code consistency. 
+This pattern is quite common inside cocos2d, and therefore in GD too. Here, we modify the `MenuLayer::init` so that it will create our label inside the layer after running the original function. If `MenuLayer::init` returns false something has gone horribly wrong and GD will probably crash but it is important to stick to the idioms for code consistency.
```

Comment:
> the `MenuLayer::init`

Functions shouldn't be referred to with the definite article.

Reply:
[Fixed](https://github.com/geode-sdk/docs/pull/5/commits/aea6bf8cc7c561156f7f9451c6a1fd16ba20590c)